portality/settings.py

```diff
@@ -1354,3 +1356,5 @@
     }
 }
 
+# report journals that discontinue in ... days (eg. 1 = tomorrow)
+DISCONTINUED_DATE_DELTA = 0
```

Comment:
Left as setting if needed to be modified in the future (even though slightly weird for 0 days = today)